README.md

```diff
@@ -1 +1,14 @@
+This is a line from R Studio. 
 # hw01
+## Isha Jain Intro
+Hi, my name is **Isha Jain** and I am third year in the college, majoring in *public policy*. On campus, I'm involved in **club soccer** and **Sliced Bread literary magazine**.
+### Locations 
+I currently live in Kampala, Uganda but I have lived in multiple cities, including:
```

Comment:
It seems that you have lived in many cities. It would be great if you could share the experience of living in different cities!